sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/CreateNewCasePage.java

```diff
@@ -44,6 +44,7 @@ public class CreateNewCasePage {
   public static final By SEX_COMBOBOX =
       By.cssSelector(".v-window [location='sex'] div[role='combobox'] div");
   public static final By SAVE_BUTTON = By.id("commit");
+  public static final By CONFIRM_BUTTON_POPUP = By.cssSelector(".popupContent #commit");
```

Comment:
duplicate of SAVE_BUTTON , the implementation for save case in in place

Reply:
the real duplicate of SAVE_BUTTON is CONTACT_CASE_SAVE_BUTTON but that was not introduced by me

Reply:
ah, it means that all confirmation popups have the same locator, ok